src/main/java/net/coderbot/iris/uniforms/WorldTimeUniforms.java

```diff
@@ -25,7 +25,8 @@ public static void addWorldTimeUniforms(UniformHolder uniforms) {
 	}
 
 	static int getWorldDayTime() {
-    		return (int) (getWorld().getTotalWorldTime() % 24000L);
+        final WorldClient world = getWorld();
+        return (int) ((useTotalWorldTime ? world.getWorldTime() : world.getTotalWorldTime()) % 24000L);
```

Comment:
nit: could you namespace this with AngelicaConfig.useTotalWorldTime?

Otherwise I got confused for a hot second as to where this was declared